scripts/add-aws-credentials.sh

```diff
@@ -12,5 +12,6 @@ cat > ~/.aws/credentials << EOL
 [default]
```

Comment:
Nit: this file refs Travis CI in two places above (lines 8 and 3). Since the PR replaces other mentions of Travis, we could do that here also.

Reply:
Fixed in 8162c58